src/plugin.yml

```diff
@@ -1,6 +1,6 @@
-name: ${project.name}
-version: ${project.version}
-description: ${project.description}
+name: Prison
```

Comment:
The name, version, and description values for `plugin.yml` are automatically set upon compilation by Maven.